src/envoy/http/authn/authenticator_base.h

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2018 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "authentication/v1alpha1/policy.pb.h"
+#include "common/common/logger.h"
+#include "server/config/network/http_connection_manager.h"
+#include "src/envoy/http/authn/context.pb.h"
```

Comment:
please remove not used headers.  I suspected http_connection_manager is not needed.

Reply:
Done